src/squid.md

```diff
@@ -0,0 +1,247 @@
+---
+title: squid
+section: 8
+---
+# NAME
+
+**squid** - HTTP web proxy caching server
+
+# SYNOPSIS
+
+**squid** [-dsCFNRSXY] [-\-foreground] [-l facility] [-f file] [-[au] port] [-k signal] [-n service]
+
+**squid** -[hv]
+
+**squid** -n service -i [-dsX] [-\-foreground] [-l facility]
+
+**squid** -n service -r [-dsX] [-\-foreground] [-l facility]
+
+**squid** -n service -O command [-ds] [-\-foreground] [-l facility]
+
```

Comment:
I assume these three are Windows-specific variants. If they have to be listed here, please move them down, after all cross-platform variants that are going to be much more frequently used.